Mathlib/Computability/DFA.lean

```diff
@@ -164,6 +164,116 @@ theorem pumping_lemma [Fintype σ] {x : List α} (hx : x ∈ M.accepts)
   have h := M.evalFrom_of_pow hb hb'
   rwa [mem_accepts, eval, evalFrom_of_append, evalFrom_of_append, h, hc]
 
+section SetClosure
+
+variable {σ' : Type v}
+
+/--
+ `M.compl` constructs a DFA for the complement of the language of `M`.
+
+ Use `Mᶜ` rather than directly using this function.
```

Comment:
```suggestion
`M.compl` constructs a DFA for the complement of the language of `M`.

Use `Mᶜ` rather than directly using this function.
```
Same below